testsuite/features/step_definitions/navigation_steps.rb

```diff
@@ -377,6 +377,8 @@
   case section
   when 'Admin'
     step %(I am authorized as "admin" with password "admin")
+  when 'Users'
```

Comment:
why do we need Users if we still use admin/admin?

Reply:
The step text let you select for which section you have rights, and if I want to access to the section Users, better have one for users, same for Images or others.